### PR TITLE
Description of cpuid and gdt.

### PR DESCRIPTION
**Pull request by RamneekSingh24**

### CODE REVIEW

**Comment by souraavv** [comment]
Reviewing this pr

**Comment by ronakladhar** [comment]
Testing

**Comment by chintansheth1711** [comment]
Testing

File: vmm-reference-gdt.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
+# [gdt.rs](https://github.com/rust-vmm/vmm-reference/blob/main/src/vm-vcpu-ref/src/x86_64/gdt.rs)
```
Comment (chintansheth1711) [inline vmm-reference-gdt.md:1]: Reviewing this PR

Comment (ronakladhar) [inline vmm-reference-gdt.md:1]: Testing

Comment (tmibvishal) [inline vmm-reference-gdt.md:1]: Testing

File: vmm-reference-gdt.md [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+
```
Comment (kshitij86) [inline vmm-reference-gdt.md:2]: Testing

**Review by souraavv — APPROVED** [review]

**Review by ronakladhar — APPROVED** [review]

**Review by chintansheth1711 — APPROVED** [review]

**Review by kshitij86 — APPROVED** [review]

**Review by codenet — APPROVED** [review]
This PR will likely conflict with other GDT PRs. Please work with author GDT PR authors to add their content too.

**Pull request merged**